### PR TITLE
doc: add description how to deploy to s390x

### PR DESCRIPTION
**Pull request by CarstenLeue**

### CODE REVIEW

File: Installing-Metacontroller-s390x.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Installing Metacontroller on an s390x k8s cluster
+
+[Metacontroller](https://github.com/metacontroller/metacontroller) is not available for the s390x platform out of the box. This [PR](https://github.com/metacontroller/metacontroller/pull/771) adds support but in the meantime it is possible to deploy metacontroller on s390x using the following recipe:
+
+## Build Metacontroller
+
+Prequisites:
+
+- an s390x virtual server (e.g. a LinuxONE box)
```
Comment (olira) [inline Installing-Metacontroller-s390x.md:9]: An s390x VSI is not required to build this component. The build can also be achieved by emulation compilation

File: Installing-Metacontroller-s390x.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    ```bash
+    cd metacontroller
+    CGO_ENABLED=0 GOOS=linux GOARCH=s390x /usr/local/go/bin/go build ./pkg/cmd/metacontroller
+    ```
+
+- build the docker image
+
+    ```bash
+    docker build . -t metacontrollerio/metacontroller:v4.7.10
```
Comment (olira) [inline Installing-Metacontroller-s390x.md:31]: this should also have the `--platform=linux/s390x` flag to make sure it's built for s390x.

File: Installing-Metacontroller-s390x.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+## Deploy the Metacontroller image into the k8s cluster (alternative A)
+
+TBD - find out how to do this
+
+## Deploy the Metacontroller image into a custom registry (alternative B)
+
+- tag the image with your registry URL
+
+    ```bash
+    docker tag metacontrollerio/metacontroller:v4.7.10 <YOUR_REGISTRY>
```
Comment (olira) [inline Installing-Metacontroller-s390x.md:45]: Does it make sense to explicitly have an example that e.g. pushes to ICCR?

**Comment by CarstenLeue** [comment]
Looks like this is not even required any more. My PR to metacontroller has been merged and `4.8.0` is building. Let's see ...

**Comment by CarstenLeue** [comment]
No more need for this workaround:

https://github.com/metacontroller/metacontroller/releases/tag/v4.8.0

**Pull request closed without merging**